src/ctapipe/containers.py

```diff
@@ -1189,6 +1189,18 @@ class MuonParametersContainer(Container):
         nan * u.deg**2,
         "MSE of the deviation of all pixels after cleaning from the ring fit",
     )
+    ring_size = Field(nan, "Number of pixels in the ring")
+    size_outside = Field(nan, "Number of pixels outside the ring")
+    num_pixels_in_ring = Field(nan, "Number of pixels in the ring")
+    mean_pixel_outside_ring = Field(nan, "Mean pixel charge outside the ring")
+    standard_dev = Field(
+        nan * u.deg,
+        "Standard deviation of the light distribution along the ring radius.",
```

Comment:
"along the ring radius" sounds like this is somehow using spatial information.

But I think this is just "of pixels on the ring", right?